src/css/_largeCSS.scss

```diff
@@ -0,0 +1,34 @@
+// Media Query for screen size over 1500px;
+@media (min-width: 1500px) {
+  header {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    height: 100px;
+    padding: 0 50px;
+    width: 94.5%;
+
+    h1 {
+      font-size: 2em;
+      margin: 0.4em 0;
+    }
+
+    li {
+      color: #d4d4d4;
+      font-size: 96%;
+      font-weight: 700;
+    }
+  }
```

Comment:
This seems like a good use of nesting - there aren't a ton of elements here to target, so rather than giving each of them a class, you just nested their selectors! nice! Seems like it saves you some characters typed overall!